data/remnant missions.txt

```diff
@@ -941,7 +941,59 @@ mission "Remnant: S-970 Regenerator"
 
 
 
-mission "Remnant: taely intro log"
+mission "Remnant: Wanderer Ramscoop"
+	name "Retrieve Wanderer Ramscoops for the Remnant"
+	description "A Remnant engineer on <planet> has offered to pay you <payment> in exchange for obtaining two ramscoops from the Wanderers for them to study."
+	minor
+	source
+		government "Remnant"
+	destination "Viminal"
+	to offer
+		has "Remnant: Technology Available: offered"
+		random < 50
+	on offer
+		require "Wanderer Ramscoop"
+		conversation
+			`Strolling into the starport, you wonder if someone from the nosy gull would be approaching you. While you have made the approach to <planet> many times, this time was differentiated by a gull that seemed to be shadowing you closely. Just looking at it, you guessed that it was probably a reconnaissance ship, or possibly a science vessel. Few other ships would carry quite so much scanning equipment.`
+			`	Looking around, you quickly notice a young Remnant trotting into the area from a different landing pad. He spots you and approaches, then clears his throat nervously "Greetings, traveler" he warbles, as if unused to singing. "We scanned you as you flew by, and noticed some unusual technology mounted to your hull. Can you tell us what they are?" With this, he pulls up his comlink and shows you an image of what appear to be your ramscoops.`
```

Comment:
`what they are` refers to "unusual technology", which is not a plural noun, so it should be `what it is` instead